.cirrus.yml

```diff
@@ -242,6 +242,8 @@ ubuntu_22_04_aarch64_stable_debug_task:
   <<: *IOX2_FFI_COMMON_DEBUG
 
 ubuntu_22_04_aarch64_beta_debug_task:
+  # TODO commented out due to limited CI time
+  only_if: false
```

Comment:
You can also temporarily deactivate cirrus ci by changing line 137 to this
```yaml
only_if: false # ( $CIRRUS_BRANCH == 'main' || ($CIRRUS_PR != '' && $CIRRUS_BASE_BRANCH == 'main'))
```

Reply:
Two more days and we have a Cirrus CI back so I didn't see the need for it. But I wanted to reduce the amount of targets - let's see if its enough for October